lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -2159,6 +2160,11 @@ export class StandardController implements IController {
             ).catch(async (e: AuthError) => {
                 // If native token acquisition fails for availability reasons fallback to web flow
                 if (e instanceof NativeAuthError && isFatalNativeAuthError(e)) {
+                    silentRequest.tokenQueryParameters =
+                        silentRequest.tokenQueryParameters || {};
+                    silentRequest.tokenQueryParameters[
+                        AADServerParamKeys.X_CLIENT_CURR_TELEM
```

Comment:
Please use [ServerTelemetryManager](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/src/telemetry/server/ServerTelemetryManager.ts) for anything related to sending server telemetry

Reply:
Unfortunately, ServerTelemetryManager is not aware of the native error context. So, there are a few options i can think of:
1. Set x_client_curr_telem platform field in StandardController and then parse it in ServerTelemetryManager
2. Add yet another extra query parameter (native_broker_error, for example) that will be added to x_client_curr_telem in ServerTelemetryManager and removed afterwards before sending request to the identity provider

Reply:
tokenQueryParameters/extraQueryParameters are really intended to be opaque to us. If we need to pass variables around we should build a separate pipeline to do so whether through new first party params or through private internal params. Option 1 sounds acceptable

Reply:
Moved the logic to ServerTelemetryManager